RequestManagementSystem/ConfigTemplate.cfg

```diff
@@ -10,17 +10,17 @@ Services
   }
   ReqProxy
   {
-    Port = 9198
+    Port = 9161
```

Comment:
Well spotted. This change of port is a bugfix: I have been asked to change the default port for ReqProxy to 9161 as 9198 was already taken.